util/env_windows_test.cc

```diff
@@ -2,12 +2,67 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file. See the AUTHORS file for names of contributors.
 
+#include <windows.h>
+
+#include <unordered_set>
+
 #include "gtest/gtest.h"
 #include "leveldb/env.h"
 #include "port/port.h"
 #include "util/env_windows_test_helper.h"
 #include "util/testutil.h"
 
+namespace {
+
+void GetOpenHandles(std::unordered_set<HANDLE>* open_handles) {
```

Comment:
(GetOpenFileDescriptors in env_posix_test.cc is a bad example here, sorry. Any chance you'd be interested in fixing that test in a separate PR? If not, it's OK, I can follow up.)

Prefer return values to out params. https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs

In this case, the function should return a std::unordered_set.

Reply:
I can't make them return non-void because I used ASSERT_* macros.